python-bindings/example.pyx

```diff
@@ -0,0 +1,12 @@
+# distutils: language=c++
+
+from pyarrow.lib cimport *
+
+
+def get_array_length(obj):
```

Comment:
We usually don't have `get_` prefix in Python. Also the example itself should be pure Python and using the `carrow` Python extension module.